guide/authentication/index.md

```diff
@@ -144,7 +144,7 @@ $body = array(
 $response = wp_remote_post( $url, array (
     'method'  => 'POST',
     'headers' => $headers,
-    'body'    =>  $data
+    'data'    =>  $body
```

Comment:
I don't think the array key should change, `wp_remote_post` takes the body as `body` last I checked?